### ISSUE TITLE
MousePosition gives wrong coordinate sometimes

### ISSUE THREAD

**Issue by PTParta** [openlayers/openlayers#13707]
Hovering the mouse over Scandinavia region the MousePosition gives a wrong coordinate sometimes. Most of the time the given coordinate is correct but looks like some reagons give a wrong coordinate. The problems manifests itself when the wrong coordinate is is put to toLonLat function (toLonLat(coordinate, "EPSG:32635")) and the result is used later. The result latitude is something over 84 or under -80 which are not even close to Scandinavia. My app is using the Scandinavia region. Don't know if the problems happens elsewhere also.

My workaround fix was to create an if statement that filters out the wrong coordinates. This workaround fixed the problem but I was wondering if others have had the issue or if there's something that could be done to fix it?

**Comment by PTParta** [comment]
No, there's no scaling of the map. I just "realized" that we are using Typescript and npm package @types for each used package. Looks like OpenLayers is most likely fine but could there be something wrong with the @types packages? I know this probably doesn't belong to OpenLayers developers but any insight would be appreciated.

**Comment by ahocevar** [comment]
Don't install `@types/ol`. OpenLayers comes with types already.
